frontend/src/globals.js

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable no-undef */
 import 'bootstrap';
+import $ from 'jquery';
```

Comment:
So this line was added in an attempt to figure out why the data table and tooltips aren't loading properly, leading to these errors. Research suggests that it has something to do with there being multiple versions of jquery installed, or jquery not being installed before datatable.

```
admin_label.js:88 Uncaught TypeError: $(...).DataTable is not a function
    at HTMLDocument.<anonymous> (admin_label.js:88)
    at mightThrow (jquery.js:3534)
    at process (jquery.js:3602)
(anonymous) @ admin_label.js:88
mightThrow @ jquery.js:3534
process @ jquery.js:3602
setTimeout (async)
jQuery.readyException @ jquery.js:3826
(anonymous) @ jquery.js:3846
mightThrow @ jquery.js:3534
process @ jquery.js:3602
setTimeout (async)
(anonymous) @ jquery.js:3640
fire @ jquery.js:3268
fireWith @ jquery.js:3398
fire @ jquery.js:3406
fire @ jquery.js:3268
fireWith @ jquery.js:3398
process @ jquery.js:3622
setTimeout (async)
(anonymous) @ jquery.js:3640
fire @ jquery.js:3268
fireWith @ jquery.js:3398
fire @ jquery.js:3406
fire @ jquery.js:3268
fireWith @ jquery.js:3398
ready @ jquery.js:3878
completed @ jquery.js:3888
admin_model.js:98 Uncaught TypeError: $(...).DataTable is not a function
    at HTMLDocument.<anonymous> (admin_model.js:98)
    at mightThrow (jquery.js:3534)
    at process (jquery.js:3602)
(anonymous) @ admin_model.js:98
mightThrow @ jquery.js:3534
process @ jquery.js:3602
setTimeout (async)
jQuery.readyException @ jquery.js:3826
(anonymous) @ jquery.js:3846
mightThrow @ jquery.js:3534
process @ jquery.js:3602
setTimeout (async)
(anonymous) @ jquery.js:3640
fire @ jquery.js:3268
fireWith @ jquery.js:3398
fire @ jquery.js:3406
fire @ jquery.js:3268
fireWith @ jquery.js:3398
process @ jquery.js:3622
setTimeout (async)
(anonymous) @ jquery.js:3640
fire @ jquery.js:3268
fireWith @ jquery.js:3398
fire @ jquery.js:3406
fire @ jquery.js:3268
fireWith @ jquery.js:3398
ready @ jquery.js:3878
completed @ jquery.js:3888
admin_irr.js:275 Uncaught TypeError: $(...).DataTable is not a function
    at HTMLDocument.<anonymous> (admin_irr.js:275)
    at mightThrow (jquery.js:3534)
    at process (jquery.js:3602)
(anonymous) @ admin_irr.js:275
mightThrow @ jquery.js:3534
process @ jquery.js:3602
setTimeout (async)
jQuery.readyException @ jquery.js:3826
(anonymous) @ jquery.js:3846
mightThrow @ jquery.js:3534
process @ jquery.js:3602
setTimeout (async)
(anonymous) @ jquery.js:3640
fire @ jquery.js:3268
fireWith @ jquery.js:3398
fire @ jquery.js:3406
fire @ jquery.js:3268
fireWith @ jquery.js:3398
process @ jquery.js:3622
setTimeout (async)
(anonymous) @ jquery.js:3640
fire @ jquery.js:3268
fireWith @ jquery.js:3398
fire @ jquery.js:3406
fire @ jquery.js:3268
fireWith @ jquery.js:3398
ready @ jquery.js:3878
completed @ jquery.js:3888
admins.js:14 Uncaught TypeError: $(...).tooltip is not a function
    at HTMLDocument.<anonymous> (admins.js:14)
    at mightThrow (jquery.js:3534)
    at process (jquery.js:3602)
```

Reply:
This was fixed by moving the imports from globals to admin